charts/sysdig/values.yaml

```diff
@@ -26,11 +26,11 @@ resources:
   # a sane defaults ones. If you have more questions or concerns, please refer
   # to Sysdig Support for more info about it
   requests:
-    cpu: 600m
-    memory: 512Mi
+    cpu: 3000m
+    memory: 3072Mi
   limits:
-    cpu: 2000m
-    memory: 1536Mi
+    cpu: 3000m
```

Comment:
why are we setting the limits and requests to be the same number

Reply:
To ensure that the agent has sufficient resources on oversubscribed nodes. It is precisely when the load on a node  approaches it's total available value that the agent's resource usage will go above requests and approach limits but if requests are considerably less than limits, then the agent will likely not get much more than requests in this scenario.